actions/src/openstack_check_actions.py

```diff
@@ -0,0 +1,447 @@
+import json
+import os
+from typing import Callable
+import logging
+import sys
+from datetime import datetime
+import requests
+from openstack_api.openstack_connection import OpenstackConnection
+from openstack_action import OpenstackAction
+from requests.auth import HTTPBasicAuth
+
+
+class CheckActions(OpenstackAction):
+
+    # pylint: disable=useless-super-delegation
+    def __init__(self, *args, **kwargs):
+        """Constructor Class"""
+        super().__init__(*args, **kwargs)
+
+    # pylint: disable=arguments-differ
+    def run(self, submodule: str, **kwargs):
+        """
+        Dynamically dispatches to the method wanted
+        """
+        func: Callable = getattr(self, submodule)
+        return func(**kwargs)
+
+    # pylint: disable=too-many-arguments, too-many-locals, no-self-use
+    def check_project(
```

Comment:
If this is an internal method the Python notation is:
```
def _check_project
```

That indicates to future readers only this file should call it. Also `check_project` is a tad too generic, what specifically does it check for?